### PR TITLE
chore(issues): create issue templates

### PR DESCRIPTION
**Pull request by kreuzerk**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (shaharkazaz) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Not sure how relevant this is here

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (shaharkazaz) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: I think we can replace `browser` and `version` with `Import conductor version:`

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
```
Comment (shaharkazaz) [inline .github/ISSUE_TEMPLATE/feature_request.md:13]: How about these sections: 
**Describe the feature**
**Describe the use case you need it for**
**Additional context**

I thought about maybe adding a checkbox for "Would you like to create a PR for it?", What do you think?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (shaharkazaz) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: **Steps to Reproduce**
1.
2.
3.

I think it's better to keep it empty, could be annoying to delete these bullets

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
```
Comment (shaharkazaz) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]: I don't think it's necessary, maybe we can replace it with "error message"?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (kreuzerk) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: Not relevant at all ;) - haven't checked correctly - will remove

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (kreuzerk) [reply .github/ISSUE_TEMPLATE/bug_report.md:28]: 👍

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 3 above)
Comment (kreuzerk) [reply .github/ISSUE_TEMPLATE/feature_request.md:13]: I like it

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (shaharkazaz) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: **To Reproduce** => **Steps to reproduce**

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+Steps to reproduce the behavior:
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - import conductor version [e.g. v2]
```
Comment (shaharkazaz) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]: missing `:`

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+**Describe the feature**
+A clear and concise description of what the problem is.
+
+**Describe the use case you need it for**
+A clear and concise description of what you need this feature for.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
```
Comment (shaharkazaz) [inline .github/ISSUE_TEMPLATE/feature_request.md:17]: `or screenshots` is it relevant in our case?

**Review by shaharkazaz — APPROVED** [review]

**Pull request merged**